privatebin/privatebin.dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM janitortechnology/ubuntu-dev
+MAINTAINER PrivateBin <support@privatebin.org>
+
+# Download deps
+RUN sudo apt-get update && \
+    sudo apt-get -y install --no-install-recommends nginx php-fpm php-gd php-sqlite3 phpunit && \
+    npm install -g mocha jsverify jsdom@9 jsdom-global@2 mime-types && \
+
+# Configure php-fpm
+    sudo mkdir -p /run/php && \
+    sudo sed -ri "s/^;daemonize = .*$/daemonize = no/" /etc/php/7.0/fpm/php-fpm.conf && \
+    sudo sed -ri "s/^(user|group) = .*$/\1 = user/" /etc/php/7.0/fpm/pool.d/www.conf && \
+
+# Configure supervisord
+    sudo sh -c "echo '[program:php-fpm]\n\
+user = user\n\
+command = sudo /usr/sbin/php-fpm7.0\n\
+autorestart = true\n\
+[program:nginx]\n\
+user = user\n\
+command = sudo /usr/sbin/nginx -g \"daemon off;\"\n\
+autorestart = true' >> /etc/supervisord.conf" && \
```

Comment:
Nit: Please feel free to move this extended Supervisor configuration into a separate `supervisord-append.conf` file, like [here](https://github.com/JanitorTechnology/dockerfiles/blob/master/discourse/supervisord-append.conf).

A less hacky way would be to import the file directly into `/etc/supervisor/conf.d/` (e.g. as `/etc/supervisor/conf.d/privatebin.conf`). We might have to add `[include]\nfiles = /etc/supervisor/conf.d/*` to our global [supervisord.conf](https://github.com/JanitorTechnology/dockerfiles/blob/master/ubuntu-dev/supervisord.conf) file for this to work.